app/controllers/flood_risk_engine/enrollments/steps_controller.rb

```diff
@@ -30,10 +30,18 @@ def update
         end
       end
 
+      def remove_exemption
+        @enrollment = Enrollment.find(params[:id])
```

Comment:
@woodpigeon  In WEX we had to implement token based lookup .. is it worth getting that in now  to save factoring calls like  find(params[:id])   later ?